pdc/apps/common/renderers.py

```diff
@@ -174,6 +180,20 @@ def replace_url(match):
         return URL_SPEC_RE.sub(replace_url, text)
 
 
+def get_ordering_field(view, method):
+    """ If the APIs have the LIST method; for the view of LIST method, add the
+    Ordering field for the users.
+    """
+    if 'list' in method and view.serializer_class:
```

Comment:
it returns None, and then, it will don't update the docs in [Query params] parts.